packages/sidecar/src/formatters/human/utils.ts

```diff
@@ -139,8 +139,8 @@ export function padLabel(label: string, width: number): string {
 }
 
 // Constants for consistent padding
-export const SOURCE_WIDTH = 7; // "browser" is the longest
-export const TYPE_WIDTH = 7; // "warning" is the longest
+export const SOURCE_WIDTH = 9; // "[BROWSER]" is the longest
```

Comment:
If you did
```

Math.max(values.map(v=>v.length));

```
or even `"[BROWSER]".length` it would be more self-explanatory and maintainable